Frameworks/UIKit/UIButton.mm

```diff
@@ -891,22 +946,18 @@ - (UIImageView*)imageView {
 }
 
 /**
- @Status Caveat
- @Notes Returns the receiving view
-*/
-- (UIView*)viewForBaselineLayout {
-    return self;
-}
-
-/**
  @Status Interoperable
 */
 - (CGSize)intrinsicContentSize {
     CGSize ret = CGSizeZero;
 
     UIImage* img = self.currentImage;
     UIEdgeInsets contentInsets = self.contentEdgeInsets;
-    CGSize textSize = [[self currentTitle] sizeWithFont:self.font];
+
+    // TODO: Should we be asking our label for its intrinsicContentSize?
```

Comment:
That makes it sound as though we shouldn't be taking the text size into account at all then.  I think in this case 'the view itself' may mean the button and its internal content/views.

---
In reply to: [100653027](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100653027) [](ancestors = 100653027)